### DISCUSSION TITLE
How do you pronounce "Vercel"?

### DISCUSSION THREAD

**Discussion by znwhite** [vercel/vercel#5340] [Help] [answered]
I always thought it was "Vare-sell" but some coworkers are making fun of me and saying that I'm making it sound "French" and that it should be pronounced "Vur-sole" and now I'm confused and sad and don't know who I am anymore.

Can you please provide a solution to this? This is mission critical.

**Comment by zackkrida** [comment, marked as answer]
It's like 'vur-sell', you can hear @rauchg pronounce it here, from the Next.js conf today: https://youtu.be/1-NzQ9ObsfM?t=281

**Reply by znwhite** [reply]
Ok so sounds like the emphasis is definitely on the second syllable. My colleagues were clearly placing emphasis on the first syllable so I will not admit a full defeat here.

Either way I will continue to pronounce it the "French" way because it sounds 3x sexier, like an expensive cologne for your laptop.

Thank you fellow Zack.

**Reply by alanmynah** [reply]
sounded like`vɜːsəl`, in case someone else was puzzled (like me) and needs it written in phonetics

upd: 
[someone pointed out](https://twitter.com/steventey/status/1400899905401462785?s=20) that it's a mash up of "versatile", "accelerate" and "excel", so it's most likely`vɜːsɛl`

**Reply by fu-sen** [reply]
Is the reading "ver" of "versatile" the same as "version"? So that more people can understand.

**Reply by alanmynah** [reply]
@fu-sen, yep, but just in case, here's the dictionary ready answer 🤣, with indicated stressed syllable. 

BE`vɜː'sɛl`
AmE `vɜr'sɛl`

You can use [this site](https://tophonetics.com/) to pick the word you think it sounds like to compare.

**Reply by fu-sen** [reply]
Thank you. I am Japanese. It may be common sense to you, but it's a word I'm unfamiliar with. Remember that there are people from all over the world here.

**Reply by alanmynah** [reply]
No worries @fu-sen, i'm not a native English speaker myself, so this is why I went for phonetics.

**Reply by fu-sen** [reply]
Yeah. This would be an answer that more people can understand. It is also important to spread "Vercel" with the correct pronunciation of "Vercel" so that more people can use it. So I dared to put in a comment to confirm this.

**Reply by warengonzaga** [reply]
Worth knowing this one haha. In my local language, we pronounced it as "Ber-Sell".

**Comment by IamManchanda** [comment]
> It's like 'vur-sell', you can hear @rauchg pronounce it here, from the Next.js conf today: https://youtu.be/1-NzQ9ObsfM?t=281

Can I call it "vur-cell" instead ... 🤔😀

**Comment by steven-tey** [comment]
The name Vercel is actually a combination of the words "versatile", "accelerate" and "excel" ([source](https://www.lexiconbranding.com/media-1/vercel-rebrand)).

So a general rule of thumb I'd use is take the "ver-" from "versatile" and the "cel" from "excel" and mash them together 😃

**Comment by ChiuMungZitAlexander** [comment]
Oops, I assume it's 'versal', like 'universal'
